src/nvim/testdir/samples/memfile_test.c

```diff
@@ -70,7 +70,7 @@ test_mf_hash(void)
 	assert(mf_hash_find(&ht, key) == NULL);
 
 	/* allocate and add new item */
-	item = (mf_hashitem_T *)lalloc_clear(sizeof(mf_hashtab_T), FALSE);
+	item = (mf_hashitem_T *)lalloc_clear(sizeof(*item), FALSE);
```

Comment:
Is there a readability or portability issue in Vim's build to not use this trick?

Reply:
I do not know about vim, but we use this in many places in our code base, so it should be fine.

Reply:
Though `lalloc_clear()` does not exist in our code base (should be `xcalloc()` ). Does this code ever run?

Reply:
It's for the cscope test only.